cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -178,6 +180,8 @@ message CassandraNode {
 
     optional int32 cassandraDaemonPid = 12;
 
+    optional string replacementForIp = 13;
```

Comment:
Is this something that will be tracked long term? Do we need to know about the replaced ip after the replacement has taken place?